src/ra_server.erl

```diff
@@ -1370,12 +1390,18 @@ overview(#{cfg := #cfg{effective_machine_module = MacMod} = Cfg,
                     last_applied,
                     cluster,
                     leader_id,
+                    voter,
                     voted_for,
                     cluster_change_permitted,
                     cluster_index_term,
                     query_index
                    ], State),
-    O = maps:merge(O0, cfg_to_map(Cfg)),
+    % implicit voter status for initial leaders
+    O1 = case O0 of
+             #{voter := _} -> O0;
+             _             -> O0#{voter => yes}
```

Comment:
remove redundant whitespace after `_` - if you want to line up the expressions use a newline.